nav-app/src/app/techniques-search/techniques-search.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component, OnInit, Input } from '@angular/core';
 import { ViewModel } from '../viewmodels.service';
-import { DataService, Technique } from '../data.service';
+import {BaseStix, DataService, Group, Mitigation, Software, Technique} from '../data.service';
 
 @Component({
   selector: 'app-techniques-search',
```

Comment:
We may want to rename this component from `techniques-search` to something more general since we can search beyond just techniques in the interface now.